project/ScalafixBuild.scala

```diff
@@ -160,62 +160,12 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
   lazy val adhocRepoCredentials = sys.props("scalafix.repository.credentials")
   lazy val isCustomRepository = adhocRepoUri != null && adhocRepoCredentials != null
 
-  override def globalSettings: Seq[Def.Setting[_]] = List(
```

Comment:
Are all of these changes necessary to fix publishing? I thought it was only publishTo that was not picked up. I don't have a strong opinion, this change is fine, just trying to learn

Reply:
yeah publishTo, scmInfo all of this has an impact on publishSigned.